app/views/orders/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1><%=@user.username%> Here are your Orders</h1>
+<%@user.orders.each do |order| %>
+  <div>
+    <p><%=order.id %> </p>
+    <p><%=order.status %> </p>
+  </div>
+  <%end %>
```

Comment:
We'll need to set a reminder to make sure these are set up eventually as a table. :)